apkid/rules/apk/packers.yara

```diff
@@ -218,6 +218,24 @@ rule dexprotector_b : packer
     not dexprotector
 }
 
+rule dexpro : packer
+{
+  // Possible newer version
```

Comment:
What does this mean? If it doesn't clarify, it should be removed.

Reply:
My bad, copy-paste from Dexprotector rule sorry. Ready to be merged